mutalyzer/GenRecord.py

```diff
@@ -312,6 +312,21 @@ def findGene(self, name) :
         return None
     #findGene
 
+    def getInfoByTranscriptID(self, transcript_id):
```

Comment:
I think it would be clearer to change the naming here. I realize you might have chosen this because of existing code and that naming is already inconsistent, but I propose to make it more explicit, e.g.,:
```python
def get_transcript_selector(self, accession):
    """
    ...

    :param str accession: Accession number for the transcript, including version.
    """
```
I don't know about *selector*, but I think *accession* is definitely clearer. I'm currently reworking much of the gene model internals and there I've started to use *index* for the "v-number". You could say that gene symbol and index together make up a selector of some sort.

What do you think?